resources/views/profile/partials/update-profile-information-form.blade.php

```diff
@@ -46,8 +46,21 @@
                 </div>
             @endif
         </div>
+        <div>
+            <header class="flex items-center">
+                <!-- Display current profile picture if it exists -->
+                <h2 class="text-lg font-medium text-gray-900"> {{ __('Add Profile Picture') }}</h2>
```

Comment:
If there is a current profile picture it should say 'Change Profile Picture' and 'Current Profile Picture' something like that